### DISCUSSION TITLE
Config.json

### DISCUSSION THREAD

**Discussion by GomSamurai** [acheong08/ChatGPT#294] [Q&A]
Hi, people! Where I must to put the config.json?

I tried in root, in src folder and everywhere and still saying: populate config.json... bla,bla,bla..

Thanks a lot!!

**Comment by Zero6992** [comment]
Just rename the file of `config.example.json` to `config.json`

**Reply by GomSamurai** [reply]
ouch, sorry man, .... it was obvious but I forget it :p thanks a lot for response
